st2api/tests/unit/controllers/v1/test_executions_simple.py

```diff
@@ -202,7 +235,7 @@ def tearDownClass(cls):
         cls.app.delete('/v1/actions/%s' % cls.action2['id'])
```

Comment:
Note: Those existing tests are old and messy - they don't use fixtures loader so we eventually need to port them.